### PR TITLE
add wired up components

### PR DESCRIPTION
**Pull request by MarquesaAsmussen**

### CODE REVIEW

**Comment by RivaD2** [comment]
@MarquesaAsmussen nice work on this! 👍 

Your PR is still open :) Not sure if you want to merge (then create a new PR for today's lab requirements) or continue to make commits for lab2 here. Either way works fine for me.

File: src/Main.js [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
+          imgsrc='http://3.bp.blogspot.com/_DBYF1AdFaHw/TE-f0cDQ24I/AAAAAAAACZg/l-FdTZ6M7z8/s1600/Unicorn_and_Narwhal_by_dinglehopper.jpg'
+          description='A unicorn and a narwhal nuzzling their horns'
+        />
+        <HornedBeast
+          title='Rhino Family'
+          imgsrc='https://images.unsplash.com/photo-1512636618879-bbe79107e9e3?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=bd9460ee6d1ddbb6b1ca7be86dfc4590&auto=format&fit=crop&w=1825&q=80'
+          description='Mother (or father) rhino with two babies'
+        />
+      </>
+    );
```
Comment (RivaD2) [inline src/Main.js:23]: Nice work on this!
  For this section, this works just fine and dandy :) To make it a little cleaner you could always build an array of objects holding the key value pairs that you want from json and then set the props of `title, imgSrc, description` etc. by using object access notation.

You guys of course will be doing more work on this anyway, just some thoughts on how you can set this up 👍 More than one way to skin a cat or something like that :)

Comment (MarquesaAsmussen) [reply src/Main.js:23]: Thanks, Riva! Yes, I am doing a json file today. Was just finishing the lab from yesterday, which apparently just wanted them hard-coded. I guess I could have just gone ahead and done the json for yesterday's too, now that I think of it... oh well, coding hind-sight is 20-20, right?

Comment (RivaD2) [reply src/Main.js:23]: You are so right here, I always think of stuff later too :) Also, good catch, I realized I got ahead of myself with that json file (after grading about 6 labs or so)  so thank you for the reminder on that one. I felt pretty darn silly after I realized that. We live and we learn :)

Nice work on this again. You are rockin' it! 💯

**Pull request merged**